### PR TITLE
[traefik] add dashboard RBAC roles

### PR DESCRIPTION
**Pull request by jr0d**
Adds view, edit, and admin roles for the traefik dashboard

### CODE REVIEW

**Comment by jr0d** [comment]
ci is having issues:
```
[17:13:22][Updating sources] VCS Root: https://github.com/mesosphere/charts#refs/heads/dev (1) (2s)
[17:13:25]
[Updating sources] Failed to perform checkout on agent: '/usr/local/bin/git clean -f -d -x' command failed.
exit code: 128
stderr: fatal: index file smaller than expected
```

File: staging/traefik/templates/dashboard-roles.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{{- if .Values.dashboardRBAC.enabled }}
```
Comment (alejandroEsc) [inline staging/traefik/templates/dashboard-roles.yaml]:
```suggestion
{{- if and .Values.dashboardRBAC.enabled .Values.dashboard.enabled }}
```

File: staging/traefik/templates/dashboard-roles.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+- nonResourceURLs:
+  - {{ .Values.dashboard.ingress.path }}
+  - {{ .Values.dashboard.ingress.path }}/*
+  verbs:
+  - get
+  - head
+  - post
+  - put
+  - delete
+{{- end}}
```
Comment (alejandroEsc) [inline staging/traefik/templates/dashboard-roles.yaml]:
```suggestion
```

File: staging/traefik/templates/dashboard-roles.yaml (hunk 1 above)
Comment (jr0d) [reply staging/traefik/templates/dashboard-roles.yaml]: :+1:

File: staging/traefik/templates/dashboard-roles.yaml (hunk 2 above)
Comment (jr0d) [reply staging/traefik/templates/dashboard-roles.yaml]: :+1:

**Pull request merged**